### DISCUSSION TITLE
Zephyr RTOS

### DISCUSSION THREAD

**Discussion by silvio-vallorani** [github0null/eide#266] [Ideas]
Please add the support for Zephyr RTOS! https://www.zephyrproject.org/

There is a good start point here: https://github.com/beriberikix/zephyr-vscode-example

**Comment by trampas** [comment]
Zephyr has it's own VSCode extensions that work well.

**Comment by github0null** [comment]
> Please add the support for Zephyr RTOS! https://www.zephyrproject.org/
> 
> There is a good start point here: https://github.com/beriberikix/zephyr-vscode-example

Zephyr has its development framework. Using this extension may not be convenient.

This extension is more suitable for projects like keil mdk (dozens of source files, not many dependencies).
